library/Kubernetes/Web/ConfigMapDetail.php

```diff
@@ -34,7 +37,20 @@ protected function assemble()
                 ]
             )),
             new Labels($this->configMap->label),
-            new Data($this->configMap->data->execute())
+            new Data($this->configMap->data->execute()),
+            new HtmlElement(
```

Comment:
As this code is repeated very often, it could be implemented as a widget like `Data`.